test/php/library/Director/Form/IcingaObjectFieldFormTest.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace Tests\Icinga\Module\Director\Field;
+
+use Icinga\Application\Test;
+use Icinga\Module\Director\Forms\IcingaObjectFieldForm;
+use Icinga\Module\Director\Objects\DirectorDatafield;
+use Icinga\Module\Director\Objects\IcingaHost;
+use Icinga\Module\Director\Test\BaseTestCase;
+use Icinga\Module\Director\Objects\IcingaCommand;
+use Icinga\Web\Request;
+
+class IcingaObjectFieldFormTest extends BaseTestCase
+{
+    /** @var string */
+    protected const COMMAND_NAME = 'icingaTestCommand';
+
+    /** @var string */
+    protected const DATAFIELD_NAME = 'dataFieldTest';
+
+    /** @var string */
+    protected const HOST_NAME = 'testHost';
+
+    /** @var ?DirectorDatafield */
+    protected $testDatafield = null;
+
+    /** @var ?IcingaCommand */
+    protected $testIcingaCommand = null;
+
+    /** @var IcingaHost */
+    private $testHost;
+
+    public function setUp(): void
+    {
+        $this->app()->setRequest(new Request());
```

Comment:
This is not required if the director's `BaseTestCase` extends Icinga Web's `BaseTestCase`. At least, works for me locally. Remember to call the parent's `setUp` then.